mcc/odin/odin/snapshot_list_test.go

```diff
@@ -35,6 +46,7 @@ type listSnapshotsCase struct {
 func (c *listSnapshotsCase) loadSnapshots(
 	svc *mockRDSClient,
 ) {
+	svc.dbSnapshots = map[string][]*rds.DBSnapshot{}
```

Comment:
This ensures no previous snapshots are being kept.
Another possible solution would be to move the mock creation within the `t.Run` part in `TestListSnapshots`, but I preferred by keeping its structure similar to other tests.